web/src/components/Link/index.tsx

```diff
@@ -47,11 +47,7 @@ const Link = memo(
             href={href}
             className={className}
             target={props.target || "_blank"}
-            rel={
```

Comment:
@igorosip0v worth being more explicit with the `external` prop to avoid misconfigurations and having a external link without noopener